src/index.css

```diff
@@ -0,0 +1,187 @@
+body {
+  margin: 0;
+  font-family:
+    -apple-system,
+    BlinkMacSystemFont,
+    'Segoe UI',
+    'Roboto',
+    'Oxygen',
+    'Ubuntu',
+    'Cantarell',
+    'Fira Sans',
+    'Droid Sans',
+    'Helvetica Neue',
+    sans-serif;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+  background-image: url(./images/cyptobg.jpeg);
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+body::before {
+  display: block;
+  content: '';
+  height: 110px;
+}
+
+code {
+  font-family:
+    source-code-pro,
+    Menlo,
+    Monaco,
+    Consolas,
+    'Courier New',
+    monospace;
+}
+
+a {
+  text-decoration: none;
+}
+
+.autoimg {
+  padding: 9px 0 9px 0;
+}
+
+.header {
+  display: flex;
+  position: fixed;
+  top: 0;
+  background-color: #002855;
+  width: 100%;
+  flex-wrap: wrap;
+  align-items: center;
+  justify-content: space-around;
+}
+
+.homepage,
+.center,
+.title {
+  color: #b1cb32;
+}
+
+.url {
+  color: #adb5bd;
+  text-decoration: underline;
+}
+
+.url:hover {
+  text-decoration: none;
+}
+
+.btn-con {
+  display: flex;
+  gap: 6px;
+  align-items: center;
+}
+
+.input {
+  display: flex;
+  justify-content: center;
+  margin: 15px 0 15px 0;
+}
+
+input {
+  width: 60%;
+  padding: 10px;
+  border: none;
+  border-radius: 10px;
+  font-size: 25px;
+  background-color: silver;
+  color: black;
+}
+
+.center {
+  text-align: center;
+}
+
+.break {
+  width: 50%;
+}
+
+.flex {
+  background-color: #001d3d;
+  display: flex;
+  align-items: center;
+  justify-content: space-around;
+  padding: 15px;
+  margin: 4px;
+  border-radius: 8px;
+}
+
+.main-container {
+  max-width: 90%;
+  margin: 0 auto;
+}
+
```

Comment:
- [ ] The details page for some of the trade companies has white patches because of few details on them. I would advise you use to make the `height` of the container `100vh` so that the white patches are hidden. 👍

![image](https://user-images.githubusercontent.com/77106962/211054460-7cc0093a-726b-4bbc-879e-a50dd931acd0.png)